WISE-Community/WISE-Client
# Remove MomentJS dependencies

## Changes
Removes all usage of MomentJS and replaces in code and template date/time formatting with Angular [formatDate](https://angular.io/api/common/formatDate) and [DatePipe](https://angular.io/api/common/DatePipe).

Closes #640.

## Test
Make sure formatted dates are displayed correctly in the following places:
- Site:
  - Footer - copyright year
  - Student run list - run start and end times, launch button (if runs hasn't started yet), last student login
  - Teacher run list - run start and end times, launch button (if runs hasn't started yet), last student login
  - Run settings dialog - last student login date/time, confirmation when changing start and end times
  - Show student info - most recent login, registration date
- Teacher Tools:
  - Notebook - save time
  - Notebook workgroup grading - latest work time
  - Workgroup component grading - save time
  - View component revisions - save time for each component state
- VLE:
  - Notebook report - save time
  - ComponentSaveSubmitButtons - latest save time (check for each component type)
  - Discussion component (class response): time of posts and replies
  - Component annotations - annotation time
  - Node component - latest component state save time (if save or submit button is enabled for node)
